envergo/analytics/forms.py

```diff
@@ -64,6 +64,7 @@ class FeedbackForm(forms.Form):
         required=False,
         label="Email ou téléphone",
         help_text="Pour vous recontacter si vous souhaitez une réponse",
+        widget=forms.EmailInput(attrs={"type": "email", "inputmode": "email"}),
```

Comment:
Je suis étonné par cette modif, il me semblait que par défaut, les widgets django embarquaient déjà le bon type. Et quand le type "email" est bien utilisé, les mobiles adaptent le clavier automatiquement. Ce n'était pas le cas ?

Reply:
Ok, en vérifiant la norme, je vois que [Inputs that require email addresses should typically use <input type="email"> instead](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode#email).

Dans la mesure ou on veut pouvoir saisir un téléphone dans ce champ, est-ce que cette modif ne risque pas de bloquer la saisie si le navigateur applique une validation en frontend ?